src/printer.js

```diff
@@ -40,14 +40,55 @@ function getLast(arr) {
   return null;
 }
 
+function getParentNodeBody(path) {
```

Comment:
do you think these fns would be better suited in `util.js`?

Reply:
also seems more generic than just `body` - maybe name something like `getParentNodeListProperty`? Naming things is hard haha

Reply:
@mgrip i think all function what working with ast should be here. But we can discussion about this and move all other function to utils. Maybe best create issue and do this in other PR :+1: